doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -590,7 +590,7 @@ Printing universes
    terms apparently identical but internally different in the Calculus of Inductive
    Constructions.
 
-.. cmd:: Print {? Sorted } Universes {? Subgraph ( {* @debug_univ_name } ) } {? @string }
+.. cmd:: Print {? Sorted } Universes {? Subgraph ( {* @debug_univ_name } ) } {? {| With | Without } Constraint Sources } {? @string }
```

Comment:
```suggestion
.. cmd:: Print {? Sorted } Universes {? Subgraph ( {* @debug_univ_name } ) } {? {| With | Without } Sources } {? @string }
```

IMO just as clear without "Constraint"  Users have to type this stuff, brevity is helpful.  Let's try not to be as wordy as COBOL :-).

Reply:
I don't really like "Sources" alone but maybe we can find a better word. "Explanations"? "Details"?

Reply:
What does the output look like when it includes sources?  That might bring some words to mind.  "Explanations" seems vague.  Possible "Verbose"?